node/core/dispute-coordinator/src/scraping/mod.rs

```diff
@@ -180,7 +170,8 @@ impl ChainScraper {
 	pub fn process_finalized_block(&mut self, finalized_block_number: &BlockNumber) {
 		// `CANDIDATE_LIFETIME_AFTER_FINALIZATION - 1` because `finalized_block_number`counts to the
 		// candidate lifetime.
-		match finalized_block_number.checked_sub(Self::CANDIDATE_LIFETIME_AFTER_FINALIZATION - 1) {
+		match finalized_block_number.checked_sub(DISPUTE_CANDIDATE_LIFETIME_AFTER_FINALIZATION - 1)
```

Comment:
nit: There's a bunch of comments like this one still referring to the old `CANDIDATE_LIFETIME_AFTER_FINALIZATION`, a quick `grep` or `rg` should show them all. 🙂